nl/mail/chrome/messenger/updates/pbUpdates.dtd

```diff
@@ -1,17 +1,17 @@
 <!ENTITY releaseNotes.label "Release Notes:">
-<!ENTITY cancelButton.label "Cancel">
-<!ENTITY installUpdateButton.label "Install Update">
-<!ENTITY newVersion.label "A new version of Postbox is available!">
+<!ENTITY cancelButton.label "Annuleren">
+<!ENTITY installUpdateButton.label "Installeer Update">
+<!ENTITY newVersion.label "Er is een nieuwe versie van Postbox beschikbaar">
 
-<!ENTITY updatingPostbox.label "Updating Postbox">
-<!ENTITY downloading.label "Downloading update...">
+<!ENTITY updatingPostbox.label "Postbox updaten">
```

Comment:
CORRECTION MIGHT BE NEEDED depending on context.
Suggestions: If this displays when Postbox is being updated, it should be: "Postbox is bezig met updaten".